metafacture-io/build.gradle

```diff
@@ -21,11 +21,12 @@ dependencies {
   api project(':metafacture-framework')
   implementation project(':metafacture-commons')
   implementation 'commons-io:commons-io:2.5'
-  implementation 'org.apache.commons:commons-compress:1.21'
+  implementation "org.apache.commons:commons-compress:${versions.commons_compress}"
   runtimeOnly 'org.tukaani:xz:1.6'
   testImplementation 'com.github.tomakehurst:wiremock-jre8:2.33.2'
-  testImplementation 'junit:junit:4.12'
-  testImplementation 'org.mockito:mockito-core:2.5.5'
-  testImplementation 'org.assertj:assertj-core:3.11.1'
-  testRuntimeOnly 'org.slf4j:slf4j-simple:1.7.21'
+  testImplementation "junit:junit:${versions.junit}"
+  testImplementation "org.mockito:mockito-core:${versions.mockito}"
+  testImplementation "org.assertj:assertj-core:${versions.assertj_core}"
+  testRuntimeOnly "org.slf4j:slf4j-api:${versions.slf4j}"
```

Comment:
Why did you add this `slf4j-api` dependency?

Reply:
This one was not removed in 8ec34cc.

Reply:
Uh, right. Removed in fd66002248f14072c341bf8bfc49e66b1ff5e860.